lib/hello-message.js

```diff
@@ -3,14 +3,10 @@ const Buffer = require('safe-buffer').Buffer;
 // Pre-serialised hello message. serial = 1
 module.exports = function() {
   return Buffer.from(
-    '6c01 0001 0000 0000 0100 0000 6d00 0000' +
-      '0101 6f00 1500 0000 2f6f 7267 2f66 7265' +
-      '6564 6573 6b74 6f70 2f44 4275 7300 0000' +
-      '0301 7300 0500 0000 4865 6c6c 6f00 0000' +
-      '0201 7300 1400 0000 6f72 672e 6672 6565' +
-      '6465 736b 746f 702e 4442 7573 0000 0000' +
-      '0601 7300 1400 0000 6f72 672e 6672 6565' +
-      '6465 736b 746f 702e 4442 7573 0000 0000'.replace(/ /g, ''),
+    `6c01 0001 0000 0000 0100 0000 6d00 00000101 6f00 1500 0000 2f6f 7267 2f66 72656564 6573 6b74 6f70 2f44 4275 7300 00000301 7300 0500 0000 4865 6c6c 6f00 00000201 7300 1400 0000 6f72 672e 6672 65656465 736b 746f 702e 4442 7573 0000 00000601 7300 1400 0000 6f72 672e 6672 6565${'6465 736b 746f 702e 4442 7573 0000 0000'.replace(
```

Comment:
should this part stay formatted as it was before?

Reply:
This will look better as template string with multiline text ( `replace` will need to take newlines into account as well and strip them )

Reply:
how do you feel about it otherwise?

Reply:
all good. ( with your enthusiasm I'm afraid we'll have static type checks soon :). Joking, it would be actually a good addition )

Reply:
I'm already trying to write a typescript definition file, but I don't actually know what I'm doing yet!

In any case though, I've solved this issue so I'm going to merge :)